dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -61,8 +68,8 @@ public Integer processContributionUpdate(UpdateLogContributionRequest updateLogC
             if (!feature.incomingIsolated()) {
                 result = contributionClient.sendLogContributionProcessed(updateLogContributionRequest);
             } else {
-                log.info("Not updating MAAT DB because feature.incomingIsolated is set to True");
-                result = 0; // avoid updating MAAT DB.
+                log.info("processContributionUpdate: not calling MAAT API sendLogContributionProcessed() because `feature.incoming-isolated=true`");
+                return 0; // avoid updating MAAT DB.
```

Comment:
As discussed, to be swapped back to `result = 0` to allow for the logging lines.